openedx/core/djangolib/tests/test_js_utils.py

```diff
@@ -65,6 +68,31 @@ def test_dump_js_escaped_json_with_custom_encoder_escapes_unsafe_html(self):
         escaped_json = dump_js_escaped_json(malicious_dict, cls=self.SampleJSONEncoder)
         self.assertEquals(expected_custom_escaped_json, escaped_json)
 
+    def test_dump_js_escaped_json_with_lazy_text(self):
+        """
+        Verify that the function dump_js_escaped_json can process a django lazy text.
+        Taking translatable text of platform name as a sample
+        """
+        lazy_text = _('Your Platform Name Here')
+        context = {
+            'sample': lazy_text,
+            'sample_dict': {
+                'sample': lazy_text,
+            },
+        }
+
+        # Verifying dumps with text
+        try:
```

Comment:
Please remove the try/catch. Shouldn't be in the tests. Generally tests should have little to no logic and goes into a straight execution path.

I suggest testing it like this:

```
lazy_text_dumped = dump_js_escaped_json(lazy_text, cls=EdxJSONEncoder)
assert lazy_text_dumped.startswith('{')
assert '"Your Platform Name Here"' in lazy_text_dumped  # Additional quotes represents JSON strings
```